classes/entity/enemy/bobomb/bobomb.tscn

```diff
@@ -93,6 +93,7 @@ extents = Vector2( 1, 1 )
 collision_layer = 4
 script = ExtResource( 7 )
 _sprite_path = NodePath("Sprites/Base")
+make_smoke = false
```

Comment:
Might be wise to call this `make_death_smoke` (or just `death_smoke`) so it's clear when the smoke in question appears.

Reply:
Done.